codespell_lib/data/dictionary.txt

```diff
@@ -1323,7 +1323,7 @@ baloon->balloon
 baloons->balloons
 balse->false
 banannas->bananas
-bandaid->band-aid
+band-aid->bandaid
```

Comment:
I think band-aid is the better spelling, bandaid is less good. We could suggest that bandaid be changed to band-aid, or just allow it. Sorry if my previous comment was unclear.

Reply:
so `bandaid->band-aid, bandaid,` ?

Reply:
I think just leave it, don't make any suggestion for 'bandaid'.